R/covariance.R

```diff
@@ -0,0 +1,31 @@
+#' Getting Number of Variance Parameters in Model
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' @param model (`list`)\cr model from glmmTMB package.
+#' @return the number of variance parameters in the model as well as one ID which has the maximum number of visits.
+#' @export
+#'
+#' @examples
+#' dat <- fev_data
+#' mod <- glmmTMB(
+#' FEV1 ~ ar1(0 + AVISIT | USUBJID),
+#' data = dat,
+#' dispformula = ~0,
+#' REML = TRUE
```

Comment:
```suggestion
#'   FEV1 ~ ar1(0 + AVISIT | USUBJID),
#'   data = fev_data,
#'   dispformula = ~0,
#'   REML = TRUE
```
(same indention rules as normally)